<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_connection</class>
 <widget class="QDialog" name="new_connection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新建服务或连接到已有服务</string>
  </property>
  <property name="windowIcon">
   <iconset resource="fiber_end_client.qrc">
    <normaloff>:/fiber_end_client/icons/new_connection.png</normaloff>:/fiber_end_client/icons/new_connection.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>320</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>IP 地址 : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ip_address">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>180</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>端口 : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_port">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>180</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>110</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>110</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="fiber_end_client.qrc"/>
 </resources>
 <connections/>
</ui>
